app/script/conversation/ConversationRepository.coffee

```diff
@@ -1815,10 +1821,21 @@ class z.conversation.ConversationRepository
   _update_user_ets: (message_et, callback) =>
     @user_repository.get_user_by_id message_et.from, (user_et) =>
       message_et.user user_et
+
       if message_et.is_member()
         @user_repository.get_users_by_id message_et.user_ids(), (user_ets) ->
           message_et.user_ets user_ets
-      else if message_et.has_asset_text()
+        return
+
+      if message_et.reactions?
```

Comment:
check for message type instead as all content message have reactions `if message_et.is_content()`